usps/main.py

```diff
@@ -1,21 +1,43 @@
-""" Views for USPS API """
-import json
-import base64
 import logging
 import ssl
+import base64
+from usps import transaction_log
 from http import HTTPStatus
-from aiohttp import ClientSession, ClientError
-from django.conf import settings
-from django.http import JsonResponse, HttpResponse
+from fastapi import FastAPI, Header
+from fastapi.responses import JSONResponse
+from pydantic import BaseModel
+from typing import Optional
+from uuid import UUID, uuid4
+from usps.config import settings
 from google.oauth2 import service_account
 from google.auth.transport.requests import Request
-from api import transaction_log
+
+app = FastAPI()
+
+
+class PersonalInfo(BaseModel):
+    uid: Optional[UUID] = None
```

Comment:
Users of GIVE should never include a uuid in their request, so the validation & docs shouldn't include it.